awx/ui_next/src/components/LaunchPrompt/LaunchPrompt.jsx

```diff
@@ -107,16 +107,22 @@ function LaunchPrompt({ config, resource = {}, onLaunch, onCancel, i18n }) {
   return (
     <Formik
       initialValues={{
-        verbosity: resource.verbosity || 0,
-        inventory: resource.summary_fields?.inventory || null,
-        credentials: resource.summary_fields?.credentials || null,
-        diff_mode: resource.diff_mode || false,
-        extra_vars: resource.extra_vars || '---',
-        job_type: resource.job_type || '',
-        job_tags: resource.job_tags || '',
-        skip_tags: resource.skip_tags || '',
-        scm_branch: resource.scm_branch || '',
-        limit: resource.limit || '',
+        verbosity: config.ask_verbosity_on_launch && (resource.verbosity || 0),
+        inventory:
+          config.ask_inventoryon_launch && resource.summary_fields?.inventory,
```

Comment:
This should be `ask_inventory_on_launch`

Reply:
@AlexSCorey yea this will cause the default inventory to not be selected on launch.  Was about to create a new comment for it when I ran across it in the UI but fixing this variable should fix it.

Before:
![before](https://user-images.githubusercontent.com/9889020/96507687-85a2de00-1227-11eb-9e38-a41fa47bd2fa.gif)

After:
![after](https://user-images.githubusercontent.com/9889020/96507694-89366500-1227-11eb-9f28-f749edded45b.gif)